Validation/DFDC/DFDC_run_J.py

```
@@ -0,0 +1,94 @@
+                TC = float(TC_match.group(1))
+                TC_list.append(TC)
+
+# Start the process
+process = subprocess.Popen(
+    ["dfdc.exe"],
+    stdin=subprocess.PIPE,
+    stdout=subprocess.PIPE,
+    stderr=subprocess.PIPE,
+    text=True,
+    bufsize=1
+)
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)